pkg/webhook/config.go

```diff
@@ -447,7 +447,7 @@ func SetConfigDefaults() {
 	viper.SetDefault("vault_image_pull_policy", string(corev1.PullIfNotPresent))
 	viper.SetDefault("vault_env_image", "ghcr.io/bank-vaults/vault-env:latest")
 	viper.SetDefault("vault_env_pull_policy", string(corev1.PullIfNotPresent))
-	viper.SetDefault("vault_ct_image", "hashicorp/consul-template:0.24.1-alpine")
+	viper.SetDefault("vault_ct_image", "hashicorp/consul-template:0.32.0")
```

Comment:
Is there an alpine version?

Reply:
Unfortunately not.
https://hub.docker.com/r/hashicorp/consul-template/tags?page=1&name=alpine

Reply:
In terms of image size, the [existing image](https://hub.docker.com/layers/hashicorp/consul-template/0.24.1-alpine/images/sha256-3f1962a18dcf9de0af25640387a63e5a32b1f5c49650a4789510ce58bdd43c74?context=explore) is 7.62 MB whereas the [new image](https://hub.docker.com/layers/hashicorp/consul-template/0.32/images/sha256-2bf5da71429412106a5f9409a55de7ac8a5081025326e3317251b5032de9c46c?context=explore) size is 7.82 MB.

Reply:
Thanks for checking!